src/Agent.Worker/TestResults/NunitResultReader.cs

```diff
@@ -8,46 +8,22 @@
 
 namespace Microsoft.VisualStudio.Services.Agent.Worker.TestResults
 {
-    public class NUnitResultReader : AgentService, IResultReader
+    internal class NUnitResultsXmlReader
```

Comment:
I would prefer to all customers to write their own extensions(DLL) by extending this